sdk/core/core-rest-pipeline/src/interfaces.ts

```diff
@@ -3,6 +3,7 @@
 
 import type { AbortSignalLike } from "@azure/abort-controller";
 import type { OperationTracingOptions } from "@azure/core-tracing";
+import type { HttpMethods } from "@azure/core-util";
```

Comment:
need to bump the min version of core-util here too

Reply:
done